docs/reference/src/language/builtins/elements.md

```diff
@@ -298,7 +298,9 @@ An `Image` can be used to represent an image loaded from a file.
 
 -   **`colorize`** (_in_ _brush_): When set, the image is used as an alpha mask and is drawn in the given color (or with the gradient).
 -   **`horizontal-alignment`** (_in_ _enum [`ImageHorizontalAlignment`](enums.md#imagehorizontalalignment)_): The horizontal alignment of the image within the element.
--   **`image-fit`** (_in_ _enum [`ImageFit`](enums.md#imagefit)_): Specifies how the source image shall be fit into the image element. Does not have any effect when used with 9 slice scaling images.
+-   **`horizontal-tiling`** (_in_ _enum [`ImageTiling`](enums.md#imagetiling)_): Whether image should be tiled on the horizontal axis.
```

Comment:
```suggestion
-   **`horizontal-tiling`** (_in_ _enum [`ImageTiling`](enums.md#imagetiling)_): Whether the image should be tiled on the horizontal axis.
```

Reply:
Just a thought: When we announce this in 1.5, it might make sense to include an example with the preview in the docs here and link to it.